includes/admin/settings/class-wc-settings-products.php

```diff
@@ -386,6 +386,16 @@ protected function get_settings_for_downloadable_section() {
 					'autoload' => false,
 				),
 
+				array(
+					'desc'          => __( 'Allow using redirect mode as a last resort', 'woocommerce' ),
+					'id'            => 'woocommerce_downloads_redirect_fallback_allowed',
+					'type'          => 'checkbox',
+					'default'       => 'no',
+					'desc_tip'      => __( 'If the "Force Downloads" or "X-Accel-Redirect/X-Sendfile" download method is selected but does not work, the system will use the "Redirect" method as a last resort.', 'woocommerce' ),
```

Comment:
Let's add a link to our support page https://docs.woocommerce.com/document/digital-downloadable-product-handling/  as well

Reply:
Updated :+1: